llvm/lib/Target/WebAssembly/WebAssemblyInstrMemory.td

```diff
@@ -171,12 +171,16 @@ defm STORE8_I64 : WebAssemblyStore<I64, "i64.store8", 0x3c>;
 defm STORE16_I64 : WebAssemblyStore<I64, "i64.store16", 0x3d>;
 defm STORE32_I64 : WebAssemblyStore<I64, "i64.store32", 0x3e>;
 
+defm STORE_F16_F32 : WebAssemblyStore<F32, "f32.store_f16", 0xfc31, [HasHalfPrecision]>;
```

Comment:
```suggestion
// Half-precision store.
defm STORE_F16_F32 :
  WebAssemblyStore<F32, "f32.store_f16", 0xfc31, [HasHalfPrecision]>;
```
How about adding one-line comment? The same for `LOAD` too.
Also 80-col wrapping.

Reply:
How about adding one for `LOAD` too?